src/spatialdata/_io/_utils.py

```diff
@@ -308,11 +343,14 @@ def save_transformations(sdata: SpatialData) -> None:
     sdata
         The SpatialData object
     """
-    from spatialdata.transformations import get_transformation, set_transformation
-
-    for element in sdata._gen_spatial_element_values():
-        transformations = get_transformation(element, get_all=True)
-        set_transformation(element, transformations, set_all=True, write_to_sdata=sdata)
+    warnings.warn(
+        "This function is deprecated and should be replaced by `SpatialData.write_transformations()` or "
+        "`SpatialData.write_metadata()`, which gives more control over which metadata to write. The execution will "
+        "conitnue by calling the former",
```

Comment:
> The execution will conitnue by calling the former

I'm not sure what this means. Also see the small typo below

```suggestion
        "`SpatialData.write_metadata()`, which gives more control over which metadata to write. The execution will "
        "continue by calling the former",
```

Reply:
"The execution will continue by calling the former" with "This function will call `SpatialData.write_transformations()`; please call this function directly."